proto/messages_test.go

```diff
@@ -370,7 +370,7 @@ func TestReadEmptyMessage(t *testing.T) {
 	}
 
 	b := buf.Bytes()
-	messages, err := readMessageSet(bytes.NewBuffer(b), int32(len(b)))
+	messages, err := readMessageSet(bytes.NewBuffer(b), int32(len(b)), -1)
```

Comment:
Same as above, I'd pass `0` instead of `-1`.

Reply:
done